pyasic/miners/backends/epic.py

```diff
@@ -323,6 +310,20 @@ async def _get_hashboards(
             except APIError:
                 pass
 
+        tuned = True
+        if web_summary is not None:
+            tuner_running = web_summary["PerpetualTune"]["Running"]
+            if tuner_running:
+                algo_info = web_summary["PerpetualTune"]["Algorithm"]
+                if algo_info.get("VoltageOptimizer") is not None:
+                    tuned = algo_info["VoltageOptimizer"].get("Optimized")
+                elif algo_info.get("BoardTune") is not None:
+                    tuned = algo_info["BoardTune"].get("Optimized")
+                else:
+                    tuned = algo_info["ChipTune"].get("Optimized")
+            # To be extra detailed, also ensure the miner is in "Mining" state
+            tuned = tuned and web_summary["Status"]["Operating State"] == "Mining"
```

Comment:
Is there a case where the board can be tuned (saved profile or such) and not be turned on?  Does it make sense to add an `enabled`/`active` flag to the hashboard data to clarify this?

Reply:
so current scheme is we don't save presets, so it is either active or inactive. for clarity perhaps active is a better term to use

Reply:
Also have to think about making it generic.  I know braiinsOS stored tuner states, but they still take a couple minutes to resume from that state.  I also dont know how this works when enabling or disabling board (a future feature I want to add), if pyasic will be able to tell if there is a valid saved state for the disabled boards.

Reply:
Things can get really complex here in the future, eventually we may want to move wattage/wattage limit to the board level, and then have a calculated property at the top level for them, since there may be cases where there is a set power limit with disabled boards (eg 3300W), and the miner has 1 disabled board, which on BOS+ would use 2200W.

Reply:
I see where your going..if thats the case then a bool isn't going to suffice here. Lets table this for now then and revisit

Reply:
I actually think 2 bools may be fine, `tuned` and `active`.  Will 100% table the wattage discussion for later, but for now i think the best interim solution is those 2 attributes...

Reply:
sounds good, let me know what you think. should we make active, tuner_active?. Just thinking if there will be confusion with "missing" or "enabled"

Reply:
There's no "enabled", I think we can't treat "active" as a pseudo enabled.